agent/tool-scripts/postprocess/kvmtrace-stop-postprocess

```diff
@@ -151,3 +178,7 @@ close(KVMTRACE_EXIT_SOURCES);
 # for $info2 (sort { $ext_int{$b} <=> $ext_int{$a} } keys(%ext_int)) {
 #	 printf "%s: %d\n", $info2, $ext_int{$info2};
 #}
+
+waitpid($zip_kallsyms_pid, 0);
+waitpid($zip_report_pid, 0);
+waitpid($zip_trace_pid, );
```

Comment:
This looks as if it would be better written in bash: It does exec/system mostly. What does perl buy us?

Reply:
Not sure what you mean.  Isn't this whole file written in `perl` already?  Not sure this commit wants to take on re-writing this code as well.